.github/workflows/build.yml

```diff
@@ -22,6 +22,8 @@ jobs:
         include:
         - os: ubuntu-latest
           cargo_flags: "--all-features"
+        - os: ubuntu-latest
+          cargo_flags: "--no-default-features"
```

Comment:
Perhaps, the default feature would be enabled by feature unification?

With my testing, `cargo build -p jj-lib --no-default-features` appears working.

Reply:
Oops, good point. We didn't have any build of just the lib crate. Should be fixed now. Tested by making a known-bad change.